test/tests.mk

```diff
@@ -88,13 +88,13 @@ INT_FLAGS ?= \
 	--istio.test.ci \
 	--istio.test.nocleanup \
 	--istio.test.kube.deploy=false \
-	--istio.test.kube.systemNamespace ${ISTIO_NS} \
-	--istio.test.kube.istioNamespace ${ISTIO_NS} \
-	--istio.test.kube.configNamespace ${ISTIO_NS} \
-	--istio.test.kube.telemetryNamespace ${ISTIO_NS} \
-	--istio.test.kube.policyNamespace ${ISTIO_NS} \
-	--istio.test.kube.ingressNamespace ${ISTIO_NS} \
-	--istio.test.kube.egressNamespace ${ISTIO_NS} \
+	--istio.test.kube.systemNamespace ${ISTIO_CONTROL_NS} \
```

Comment:
not `{ISTIO_SYSTEM_NS}`?

Reply:
discovery service is installed in `${ISTIO_CONTROL_NS}`